flink-streaming-java/src/main/java/org/apache/flink/streaming/api/environment/StreamExecutionEnvironment.java

```diff
@@ -958,8 +959,7 @@ public TimeCharacteristic getStreamTimeCharacteristic() {
 	 * @return The data stream that represents the data read from the given file as text lines
 	 */
 	public DataStreamSource<String> readTextFile(String filePath, String charsetName) {
-		Preconditions.checkNotNull(filePath, "The file path must not be null.");
-		Preconditions.checkNotNull(filePath.isEmpty(), "The file path must not be empty.");
+		Preconditions.checkArgument(!StringUtils.isNullOrWhitespaceOnly(filePath), "The file path must not be blank.");
```

Comment:
How about change the error message to `The file path must not be null or blank.`

Reply:
> Good catch and thanks for your contribution. Only two minor comments left, please consider to modify the error message.

ok, have modified.